src/diffusers/pipelines/ltx/pipeline_ltx_i2v_long_multi_prompt.py

```diff
@@ -0,0 +1,1256 @@
+            for i in range(len(texts)):
+                if texts[i] == "":
+                    texts[i] = last
+                else:
+                    last = texts[i]
+            return texts
+
+        # bar-split mode
+        if isinstance(prompt, str):
+            parts = [p.strip() for p in prompt.split("|")]
+        else:
+            parts = prompt
+        parts = [p for p in parts if p is not None]
+        return parts
+
+    def batch_normalize(self, latents, reference, factor):
+        """
+        Batch AdaIN-like normalization for latents in dict format (ComfyUI-compatible).
+        Args:
+            latents: dict containing "samples" shaped [B, C, F, H, W]
+            reference: dict containing "samples" used to compute target stats
+            factor: float in [0, 1]; 0 = no change, 1 = full match to reference
+        Returns:
+            Tuple[dict]: a single-element tuple with the updated latents dict.
+        Notes:
+            - Operates channel-wise across all temporal and spatial dims (T, H, W).
+            - Uses torch.std_mean for numerical stability.
+            - Final samples are linearly interpolated with the original via `factor`.
+        """
+        latents_copy = copy.deepcopy(latents)
+        t = latents_copy["samples"]  #  B x C x F x H x W
+
+        for i in range(t.size(0)):  # batch
+            for c in range(t.size(1)):  # channel
+                r_sd, r_mean = torch.std_mean(reference["samples"][i, c], dim=None)  # index by original dim order
+                i_sd, i_mean = torch.std_mean(t[i, c], dim=None)
+
+                t[i, c] = ((t[i, c] - i_mean) / i_sd) * r_sd + r_mean
+
+        latents_copy["samples"] = torch.lerp(latents["samples"], t, factor)
+        return (latents_copy,)
+
+    def apply_cfg(
+        self,
+        noise_pred: torch.Tensor,
+        noise_pred_uncond: Optional[torch.Tensor],
+        noise_pred_text: Optional[torch.Tensor],
+        guidance_scale: float,
+        guidance_rescale: float,
+    ) -> torch.Tensor:
+        """
+        Unified classifier-free guidance (CFG) composition.
+
+        Args:
+          noise_pred: Tensor. Base prediction to return if CFG components are None.
+          noise_pred_uncond: Optional[Tensor]. Unconditional prediction.
+          noise_pred_text: Optional[Tensor]. Text-conditional prediction.
+          guidance_scale: float. CFG scale w.
+          guidance_rescale: float. Optional rescale to mitigate overexposure (see `rescale_noise_cfg`).
+
+        Returns:
+          Tensor: combined = uncond + w * (text - uncond), optionally rescaled.
+        """
+        if noise_pred_uncond is None or noise_pred_text is None:
+            return noise_pred
+        combined = noise_pred_uncond + guidance_scale * (noise_pred_text - noise_pred_uncond)
+        if guidance_rescale > 0:
+            combined = rescale_noise_cfg(combined, noise_pred_text, guidance_rescale=guidance_rescale)
+        return combined
```

Comment:
```suggestion
```
let's not have this method, instead, having the logic directly implemented inside pipeline